sdk/purview/azure-purview-workflow/azure/purview/workflow/aio/operations/_operations.py

```diff
@@ -165,17 +166,15 @@ async def extract_data(pipeline_response):
             return deserialized.get("nextLink") or None, AsyncList(list_of_elem)
 
         async def get_next(next_link=None):
-            request = prepare_request(next_link)
+            _request = prepare_request(next_link)
 
             _stream = False
             pipeline_response: PipelineResponse = await self._client._pipeline.run(  # pylint: disable=protected-access
-                request, stream=_stream, **kwargs
+                _request, stream=_stream, **kwargs
             )
             response = pipeline_response.http_response
 
             if response.status_code not in [200]:
-                if _stream:
-                    await response.read()  # Load the body in memory and close the socket
```

Comment:
Is this now happening somewhere else?

Reply:
@iscai-msft do you know the reason for the change in the generated operations here?

Reply:
hmm that's interesting, let me take a look at this. We had removed this logic for stream-only operations, but I think this got overshot. Let me work on this today

Reply:
@annatisch @kristapratico @iscai-msft It is expected since _stream is always "False":

![image](https://github.com/Azure/azure-sdk-for-python/assets/70930885/1be69b02-341e-4e3e-aeb3-96d0b2e361f9)

Reply:
forgot to respond in this issue, but I believe this issue got resolved, is that correct @kristapratico?